wsd/Storage.hpp

```diff
@@ -420,6 +420,13 @@ class WopiStorage : public StorageBase
         bool getSupportsRename() const { return _supportsRename; }
         bool getSupportsLocks() const { return _supportsLocks; }
         bool getUserCanRename() const { return _userCanRename; }
+        bool isFreemiumUser() const
+        {
+#ifdef ENABLE_FREEMIUM
+            return _isFreemiumUser;
+#endif
+            return false;
+        }
```

Comment:
Surely this has nothing to do with storage. Why is storage involved at all?

Reply:
removed. was left over